updatecli/updatecli.d/charts/accountapp.yaml

```diff
@@ -29,7 +29,6 @@ targets:
       file: clusters/publick8s.yaml
       engine: yamlpath
       key: $.releases[?(@.name == 'accountapp')].version
-    scmid: default
```

Comment:
Sounds like a mistake: you should not remove this instructions. Otherwise the target would never be applied to the git repository

Reply:
👍